index.html

```diff
@@ -17,178 +21,339 @@
-    <script type="text/javascript" src="assets/lib/bootstrap/dist/js/bootstrap.min.js"></script>
-    <script type="text/javascript" src="assets/lib/mustache/mustache.min.js"></script>
+    <script
+      type="text/javascript"
+      src="assets/lib/jquery/dist/jquery.min.js"
+    ></script>
+    <script
+      type="text/javascript"
+      src="assets/lib/jquery-easing-original/jquery.easing.min.js"
+    ></script>
+    <script
+      type="text/javascript"
+      src="assets/lib/bootstrap/dist/js/bootstrap.min.js"
+    ></script>
+    <script
+      type="text/javascript"
+      src="assets/lib/mustache/mustache.min.js"
+    ></script>
     <script type="text/javascript" src="assets/js/script.js"></script>
     <script type="text/javascript" src="/assets/js/officers.js"></script>
     <script type="text/javascript" src="assets/js/alumni.js"></script>
-</head>
+  </head>
 
-<body id="page-top" data-spy="scroll" data-target=".navbar-fixed-top">
+  <body id="page-top" data-spy="scroll" data-target=".navbar-fixed-top">
     <div id="navbar"></div>
     <div id="fb-root"></div>
-<script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v14.0" nonce="3ssviHgQ"></script>
+    <script
+      async
+      defer
+      crossorigin="anonymous"
+      src="https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v14.0"
+      nonce="3ssviHgQ"
+    ></script>
     <script>
-        $(function() {
-            $("#navbar").load("/components/navbar.html");
-        });
+      $(function () {
+        $("#navbar").load("/components/navbar.html");
+      });
     </script>
 
-    <div class="row" id="home">
-        <div class="home-header-container">
-            <h2 class="home-subheader">Women in Computer Science at UIUC</h2>
+    <!--main header carousel-->
+    <div id="mainCarousel" class="carousel slide" data-ride="carousel">
+      <!--indicators-->
+      <ol class="carousel-indicators">
+        <li data-target="#mainCarousel" data-slide-to="0" class="active"></li>
+        <li data-target="#mainCarousel" data-slide-to="1"></li>
+        <li data-target="#mainCarousel" data-slide-to="2"></li>
+        <li data-target="#mainCarousel" data-slide-to="3"></li>
+      </ol>
+      <!--images in carousel-->
+      <div class="carousel-inner">
+        <div class="item active">
+          <img
+            src="/assets/img/events/wcs bits and bytes.png"
+            class="carouselImageStyling"
+            alt="bits & bytes"
+          />
+          <div class="carousel-caption">
+            <h2 class="carouselText">Women in Computer Science at UIUC</h2>
+          </div>
+        </div>
+        <div class="item">
+          <img
+            src="/assets/img/events/wcs google workshop.png"
+            class="carouselImageStyling"
+            alt="google"
+          />
+          <div class="carousel-caption">
+            <h2 class="carouselText">Women in Computer Science at UIUC</h2>
```

Comment:
can we overlay the "Women in Computer Science at UIUC" on top of the entire carousel instead of having it be a part of the carousel? you can look into absolute positioning to make this happen ^^